README.md

```diff
@@ -36,25 +36,33 @@ just deploy-astria-local
 
 ### Configuring Funding of Geth
 
-By default running this local node will fund a wallet address `0xaC21B97d35Bf75A7dAb16f35b111a50e78A72F30` which you can add to your preferred wallet using the private key in `kubernetes/geth/key/private_key.txt`. This account should never be used for anything but test transactions.
+By default, running this local node will fund a wallet address `0xaC21B97d35Bf75A7dAb16f35b111a50e78A72F30`, which you can add to your preferred wallet using the private key in `kubernetes/geth/key/private_key.txt`. This account should never be used for anything but test transactions.
 
-To change the wallet account which receives funds alter the `alloc` section of `kubernetes/geth/genesis/geth-genesis.json`
+To change the wallet account which receives funds, alter the `alloc` section of `kubernetes/geth/genesis/geth-genesis.json`
+
+### Faucet
+
+The faucet is reachable at http://executor.astria.localdev.me/faucet. 
+
+By default, the faucet is funded by the account that is funded during geth genesis. This is configured by using the private key of the funded account in `start-faucet.sh`. This key is defined in `kubernetes/faucet/config-maps.yml` and is identical to the key in `kubernetes/geth/key/private_key.txt`.
 
 ### Connecting Metamask
 
 * add custom network
-  * network name: `astria-local`
-  * rpc url: `http://executor.astria.localdev.me`
-  * chain id: `912559`
-  * currency symbol: `ETH`
+    * network name: `astria-local`
+    * rpc url: `http://executor.astria.localdev.me`
+    * chain id: `912559`
+    * currency symbol: `ETH`
+
 
 ## Deployments and Containers
 
-| Deployment | Containers |
-| --- | --- |
+| Deployment       | Containers                    |
+|------------------|-------------------------------|
```

Comment:
Is this autoformatting?

Reply:
Yeah, from pycharm. I left it because it does make it more readable if you're reading the markdown without a parser.